packages/testing/src/testProxy.ts

```diff
@@ -0,0 +1,71 @@
+import { getHeaderByKey, ServerApp, ServerProxy, ServerProxyRequest, ServerProxyResponse } from '@marblejs/proxy';
+import { HttpStatus } from '@marblejs/core';
+import { defaultTestProxyOptions, TestProxyOptions, TestRequest, TestResponse } from './testProxy.options';
+
+const normalizeBody = (contentType: string, body: any): Buffer => {
```

Comment:
You can reuse `defaultParser` from `@marblejs/middleware-body` 🤔

Reply:
I didn't want to force dependency on body middleware.